README.md

```diff
@@ -63,21 +63,20 @@ _* Not verified. Anyone who is able to verify that this driver works or not is e
 #### Prerequisites
 
 * Internet connection
-* [ROS](http://wiki.ros.org/ROS/Installation) _Indigo_ or _Jade_
-* Ubuntu packages: `python-rosdep`, `python-catkin-tools`
+* [ROS](http://wiki.ros.org/ROS/Installation) _Foxy_
```

Comment:
This installation link should point to https://index.ros.org/doc/ros2/Installation/#installationguide

Reply:
Fixed.